### PR TITLE
Create VM nits

### PR DESCRIPTION
**Pull request by ezr-ondrej**

### CODE REVIEW

File: test/unit/foreman_google/google_compute_adapter_test.rb [hunk 1]
```diff
@@ -113,7 +113,7 @@ class GoogleComputeAdapterTest < GoogleTestCase
 
         args = {
           name: 'foreman-test',
-          machine_type: 'zones/us-east1-b/machineTypes/e2-micro',
```
Comment (stejskalleos) [inline test/unit/foreman_google/google_compute_adapter_test.rb]: This is correct, see https://github.com/stejskalleos/foreman_google/blob/develop/app/models/foreman_google/google_compute.rb#L86

File: app/models/foreman_google/google_compute.rb [hunk 2]
```diff
@@ -108,7 +108,7 @@ def load
     end
 
     def parameterize_name(name)
-      name&.parameterize || "foreman_#{Time.now.to_i}"
+      name&.parameterize || "foreman-#{Time.now.to_i}"
```
Comment (stejskalleos) [inline app/models/foreman_google/google_compute.rb:111]: Tests needs to be updated as well:
```
Failure:
ForemanGoogle::GoogleComputeTest::#name & #hostname#test_0001_default value [/home/runner/work/foreman_google/foreman_google/foreman_google/test/models/foreman_google/google_compute_test.rb:65]
Minitest::Assertion: Expected "foreman-1649099387" to include "foreman_".
```

**Review by stejskalleos — CHANGES_REQUESTED** [review]

File: test/unit/foreman_google/google_compute_adapter_test.rb (hunk 1 above)
Comment (ezr-ondrej) [reply test/unit/foreman_google/google_compute_adapter_test.rb]: oh right, I thouth these are args for google_compute initialize, my bad!

**Comment by stejskalleos** [comment]
:heavy_check_mark: LGTM, thanks @ezr-ondrej!

**Pull request merged**